docs/pages/world-explorer.mdx

```diff
@@ -0,0 +1,167 @@
+import { Tabs, Tab } from "nextra/components";
+import { CollapseCode } from "../components/CollapseCode";
+
+# World Explorer
+
+World Explorer is a GUI tool designed for visually exploring and manipulating the state of worlds.
+It is available in the MUD TypeScript templates starting with version 2.2.
+
+## Usage
+
+The URL for World Explorer is [http://localhost:13690](http://localhost:13690).
```

Comment:
Don't know if it makes sense to show and link this before even telling folks how to run it. It'll be confusing if folks open this up to a non working page.